src/contact.ts

```diff
@@ -477,6 +477,37 @@ export class Contact implements Sayable {
    */
   public alias(empty: null): Promise<boolean>
 
+  /**
+   * GET / SET / DELETE the alias for a contact
+   *
+   * @param {(none | string | null)} newAlias , 
+   * @returns {(string | null | Promise<boolean>)}
+   *
+   * @example GET the alias for a contact
+   * ```ts
+   * const alias = contact.alias()  //@returns {(string | null)}
+   * ```
+   * 
+   * @example SET the alias for a contact
+   * ```ts
+   * const ret = await contact.alias('lijiarui')  //@returns {Promise<boolean>}
+   * if (ret) {
+   *   console.log(`change ${contact.name()}'s alias successfully!`)
+   * } else {
+   *   console.error('failed to change ${contact.name()}'s alias!')
+   * }
+   * ```
+   * 
+   * @example DELETE the alias for a contact
+   * ```ts
+   * const ret = await contact.alias(null)  //@returns {Promise<boolean>}
```

Comment:
Get rid of the comments in example code too.

Example code should be very easy to understand without any comments. Or there's must be something wrong.